### PR TITLE
rewrite PRO functions to query GraphQL

### PR DESCRIPTION
**Pull request by benubah**
This PR rewrites `get_pro_groups()` and `get_pro_events()` to query the new GraphQL end-points.

### CODE REVIEW

**Comment by benubah** [comment]
@maelle Wondering how you got the old `get_pro()` functions to pass unit tests using an encrypted `meetup-pro-token` and GitHub secrets? Could you also take a look here? `R CMD check` is failing because of this. Thanks!

File: R/get_pro.R [hunk 1]
```diff
@@ -5,77 +5,96 @@
...
-#' #' @template verbose
 #' #'
 #' #' @references
-#' #' \url{https://www.meetup.com/meetup_api/docs/pro/:urlname/groups/}
-#' #' \url{https://www.meetup.com/meetup_api/docs/:urlname/events/#list}
+#' #' \url{https://www.meetup.com/api/schema/#ProNetwork}
 #' #'
 #' #' @examples
-#' #' \dontrun{
+#' #' try(
```
Comment (drmowinckels) [inline R/get_pro.R]:
```suggestion
#' #' \dontrun(
```

needs to be wraped in dontrun for when we submit to CRAN in the future

File: R/get_pro.R [hunk 2]
```diff
@@ -5,77 +5,96 @@
 #' #'
```
Comment (drmowinckels) [inline R/get_pro.R]: Would be great with a description of the `status` parameter options that can be provided, so users know what the options are.

**Review by drmowinckels — CHANGES_REQUESTED** [review]
Great stuff. Just some minor changed I'd like. 
I'll get working on the token generation, its been a while, but I'm sure I can figure it out again.

**Pull request merged**